### PR TITLE
APEXMALHAR-2111 Projection Operator config changes

### PR DESCRIPTION
**Pull request by pradeepdalvi**
- Change params to use List instead of comma-separated field names
- Mark Operator as Evolving
- Add projection operator exception in pom.xml for japicmp-maven-plugin

### CODE REVIEW

**Comment by pradeepdalvi** [comment]
@chinmaykolhatkar please review.

File: library/src/main/java/com/datatorrent/lib/projection/ProjectionOperator.java [hunk 1]
```diff
@@ -308,5 +303,45 @@ private void handleProjection(Object t)
     }
   }
 
+  /**
+   * set selectFields, a list of fields to be selected from incoming POJO
+   *
+   * @param selectFields List of fields from POJO to be selected
+   */
+  public void setSelectFields(List<String> selectFields)
```
Comment (chinmaykolhatkar) [inline library/src/main/java/com/datatorrent/lib/projection/ProjectionOperator.java:313]: Now that we have proper configuration for selectFields, can you please add @useSchema and @description javadoc metatag. You can refer to POJOEnricher setIncludeFields setter method for this.

File: library/src/main/java/com/datatorrent/lib/projection/ProjectionOperator.java [hunk 2]
```diff
@@ -308,5 +303,45 @@ private void handleProjection(Object t)
...
+  {
+    return selectFields;
+  }
+
+    /**
+   * set dropFields, a list of fields to be dropped from incoming POJO
+   *
+   * @param dropFields List of fields from POJO to be selected
+   */
+  public void setDropFields(List<String> dropFields)
```
Comment (chinmaykolhatkar) [inline library/src/main/java/com/datatorrent/lib/projection/ProjectionOperator.java:335]: Same as selectFields. Please add @useSchema and @description metatag.

**Comment by chinmaykolhatkar** [comment]
Looks fine to me. I'll wait for any other comments for couple days and then merge the changes.

**Pull request merged**